rfcs/20210211-bnorm-backward_data-and-backward_weights/README.md

```diff
@@ -0,0 +1,61 @@
+# Proposal for support backward data (actually not supported for scale and shift) and backward weights in batch normalization.
+
+## Problem and motivation
+
+Currently in OneDNN library batch normalization support following propagation 
+kinds: forward_inference, forward_training, backward and backward_data (only 
+without scale and shift). Tensorflow team requested the possibility to use 
+backward_data with scale and shift. It is worth adding backward_weights 
```

Comment:
> Tensorflow team requested the possibility to use 
backward_data with scale and shift.

Do we know what the problem is to use `dnnl_backward` in such case? It does what is requested.

Reply:
Still waiting for this information.

Reply:
Ok, I received information, that they saw the crash while running a customer model but they do not recall now which model it was. They are still in favor of this change.

Reply:
Doesn't sound right to me as a primary motivator. I'd ask them to reproduce the issue and bring us VERBOSE output so that we could take a look since *there's no need in API extension to get what they want*.